Dokumentationen/test/test_cases.adoc

```diff
@@ -315,13 +315,17 @@ a| * gefilterte, inkl. aller gebuchten Experimente, Dozentenwoche
 
 |===
 
-|*Beschreibung* |
+|*Beschreibung* |Es muss sichergestellt werden, dass der Admin keinen Zugriff auf das Dozenten-Journal hat, die Dozenten nur ihr persönliches Journal einsehen dürfen und nur der Admin die Statistik abrufen kann. Auch dürfen die Dozenten nur ihre eigene Dozentenwoche abrufen. 
 
 |*Vorbedingungen und benötigte Daten*
-a| * 
+a| * Admin ist angemeldet
+* ein Dozent ist mit validen Zugangsdaten angemeldet
+
 
 |*Nachbedingung*
-a| *
```

Comment:
evlt. in der Vergangenheit schreiben. Das spielt zwar sicherlich keine große Rolle, würde aber für den letzten Touch Einheitlichkeit sorgen. :D